src/messaging/message.asserts.ts

```diff
@@ -138,3 +138,12 @@ export function assertBeeUrl(url: string): asserts url {
     throw new Error('Blossom: Invalid Bee URL')
   }
 }
+
+export function isConvertedUint8Array(data: unknown): data is Record<string, number> {
+  return (
+    typeof data === 'object' &&
+    Object.entries(data).every(
+      ([key, value]) => !Number.isNaN(key) && Number.isInteger(Number(key)) && Number.isInteger(value),
+    )
```

Comment:
Could you rewrite this into a lighter function? we shouldn't check each of its elements because it can have many